src/lib/blake2/blake2.ml

```diff
@@ -31,22 +31,25 @@ module Make () = struct
       module T = struct
         type t = T1.t
         [@@deriving version {asserted; unnumbered}, hash, sexp, compare]
-
-        [%%define_locally
-        T1.(to_raw_string, digest_string, to_hex)]
-
-        include Binable.Of_stringable (T1)
-        include Hashable.Make (T1)
-        include Comparable.Make (T1)
       end
 
       include T
+      include Binable.Of_stringable (T1)
+      include Hashable.Make (T1)
```

Comment:
Do we need these to be in the version? I would feel more comfortable if `Hashable` and `Comparable`  here were only in the main module.

Reply:
I'd left them in because the signature required them. But removing them from the signature didn't break anything, so now I've removed them from the implementation.